src/codegen/property_body.rs

```diff
@@ -138,7 +138,10 @@ impl Builder {
         } else {
             ".unwrap()"
         };
-        body.push(Chunk::Custom(format!("value.get(){}", unwrap)));
+        body.push(Chunk::Custom(format!(
+            "value.get().expect(\"Return Value for property getter\"){}",
```

Comment:
Maybe include the property name here and why this fails

Reply:
Ok for name. However, it though the [error would be displayed in the panic message](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect), so we only needed to write the context in the `expect` message.

Reply:
true

Reply:
... but unfortunately, `expect` seems to be using the `Debug` version, not the `Display` one: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=7da532d4cd23ff02e61fb7085df8bab8

So we would get (example from `gio`):

```
thread 'main' panicked at 'Return Value for property `from-charset` getter: GetError { actual: guint, requested: gboolean }'
```

We loose the `TypeMismatch` part even though it is kind of obvious from the fields. So I guess the `enum` version could be applicable finally:

``` rust
pub enum GetError {
    TypeMismatch { actual: Type, requested: Type },
}
```

Reply:
:+1:

Reply:
OTOH, how does it work for `TryFromIntError` then? That should have the same problem you have here.

Reply:
Not much details: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0c0d17053f263ea0bb6b2c6cb2340847

Reply:
Let's keep consistent with that then, I guess

Reply:
I.e. it's not our problem but something that should be solved in `expect()` and `unwrap()`

Reply:
All right!